sql-db/multiple-pus/src/main/java/io/quarkus/ts/openshift/sqldb/multiplepus/FruitResource.java

```diff
@@ -0,0 +1,86 @@
+package io.quarkus.ts.openshift.sqldb.multiplepus;
+
+import io.quarkus.hibernate.orm.PersistenceUnit;
+import io.quarkus.panache.common.Sort;
+import io.quarkus.ts.openshift.sqldb.multiplepus.model.fruit.Fruit;
+
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.transaction.Transactional;
+import javax.validation.Valid;
+import javax.ws.rs.ClientErrorException;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import java.util.List;
+
+@Path("fruit")
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+public class FruitResource {
+
+    @Inject
+    @PersistenceUnit("fruits")
+    EntityManager entityManager;
+
+    @GET
+    public List<Fruit> getAll() {
+        return Fruit.listAll(Sort.by("name"));
+    }
+    
+    @GET
+    @Path("/{id}")
+    public Fruit get(@PathParam("id") Long id) {
+        Fruit fruit = Fruit.findById(id);
```

Comment:
Consider using a final local variable. Or replace this statement with an Optional
Example:
Optional.ofNullable(Vegetable.findById(id)).orElseThrow(()->new NotFoundException("vegetable '" + id + "' not found"));

Reply:
The existing code is just fine.

Reply:
this is a suggestion!